xbmc/utils/test/TestLabelFormatter.cpp

```diff
@@ -51,7 +51,7 @@ TEST_F(TestLabelFormatter, FormatLabel)
 TEST_F(TestLabelFormatter, FormatLabel2)
 {
   XFILE::CFile *tmpfile;
-  std::string tmpfilepath, destpath;
+  std::string tmpfilepath;
```

Comment:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-variables

Use CamelCase. Type prefixing (Systems Hungarian notation) is discouraged.